packages/pebble-web/src/components/__tests__/__snapshots__/phonenumberinput.test.tsx.snap

```diff
@@ -1,7 +1,7 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
 exports[`Component: Select default snapshot 1`] = `
-.emotion-12 {
+.emotion-10 {
```

Comment:
I did not understand what is that you are trying to do , like are you trying to downgrade the emotion version or something.

Reply:
`.snap` files are generated by [jest snapshot tests](https://jestjs.io/docs/snapshot-testing)